packages/broker/src/broker.ts

```diff
@@ -19,19 +24,28 @@ export interface Broker {
     stop: () => Promise<unknown>
 }
 
+const applyPluginClientConfigs = (plugins: Plugin<any>[], clientConfig: StrictConfig['client']) => {
+    plugins.forEach((plugin) => {
+        plugin.getClientConfig().forEach((item) => {
```

Comment:
I would prefer for-loops.
```suggestion
    plugins.forEach((plugin) => {
        plugin.getClientConfig().forEach((item) => {
```